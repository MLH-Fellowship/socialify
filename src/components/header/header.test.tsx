import { render } from '@testing-library/react'

import Header from './header'

test('Header renders', () => {
  const { container } = render(<Header />)
  const header = container.firstElementChild!

  expect(header).toMatchSnapshot()
})
