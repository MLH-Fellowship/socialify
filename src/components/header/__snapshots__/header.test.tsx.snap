// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<header
  class="jsx-83ad07ff6c7c3889"
>
  <div
    class="ant-page-header header ant-page-header-ghost"
    style="width: 95%;"
  >
    <div
      class="ant-page-header-heading"
    >
      <div
        class="ant-page-header-heading-left"
      >
        <div
          class="ant-page-header-back"
        >
          <div
            aria-label="Back"
            class="ant-page-header-back-button"
            role="button"
            style="border: 0px; background: transparent; padding: 0px; line-height: inherit; display: inline-block;"
            tabindex="0"
          >
            <span
              aria-label="home"
              class="anticon anticon-home"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="home"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M946.5 505L560.1 118.8l-25.9-25.9a31.5 31.5 0 00-44.4 0L77.5 505a63.9 63.9 0 00-18.8 46c.4 35.2 29.7 63.3 64.9 63.3h42.5V940h691.8V614.3h43.4c17.1 0 33.2-6.7 45.3-18.8a63.6 63.6 0 0018.7-45.3c0-17-6.7-33.1-18.8-45.2zM568 868H456V664h112v204zm217.9-325.7V868H632V640c0-22.1-17.9-40-40-40H432c-22.1 0-40 17.9-40 40v228H238.1V542.3h-96l370-369.7 23.1 23.1L882 542.3h-96.1z"
                />
              </svg>
            </span>
          </div>
        </div>
        <span
          class="ant-page-header-heading-title"
          title="GitHub Socialify"
        >
          GitHub Socialify
        </span>
      </div>
      <span
        class="ant-page-header-heading-extra"
      >
        <div
          class="ant-space ant-space-horizontal ant-space-align-center"
        >
          <div
            class="ant-space-item"
          >
            <div
              class="jsx-83ad07ff6c7c3889"
            >
              <a
                class="jsx-83ad07ff6c7c3889"
                href="https://www.producthunt.com/posts/socialify?utm_source=badge-featured&utm_medium=badge&utm_souce=badge-socialify"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt="Socialify - 💞 Socialify your project. 🌐 Share with the world! | Product Hunt"
                  class="jsx-83ad07ff6c7c3889"
                  height="54"
                  src="https://api.producthunt.com/widgets/embed-image/v1/featured.svg?post_id=271993&theme=light"
                  style="width: 250px; height: 54px;"
                  width="250"
                />
              </a>
            </div>
          </div>
        </div>
      </span>
    </div>
  </div>
  <a
    aria-label="View source on GitHub"
    class="ant-typography github-corner"
    href="https://github.com/wei/socialify"
  >
    <svg
      class="jsx-83ad07ff6c7c3889 github-svg"
      viewBox="0 0 250 250"
    >
      <path
        class="jsx-83ad07ff6c7c3889"
        d="M0,0 L115,115 L130,115 L142,142 L250,250 L250,0 Z"
      />
      <path
        class="jsx-83ad07ff6c7c3889 octo-arm"
        d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2"
        fill="currentColor"
        style="transform-origin: 130px 106px;"
      />
      <path
        class="jsx-83ad07ff6c7c3889 octo-body"
        d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z"
        fill="currentColor"
      />
    </svg>
  </a>
</header>
`;
